src/live.asp.net/wwwroot/js/site.js

```diff
@@ -1,10 +1,10 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved. 
+﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 (function (window, undefined) {
-    "use strict";
+    'use strict';
```

Comment:
this is a jshint violation.  You can add a `.jshintrc` file and set the options. For the single quote
```
"quotmark"      : false,
```